README.md

```diff
@@ -25,3 +26,22 @@ while  not board.is_game_over() and  not board.can_claim_draw():
 
 engine.quit()
 ```
+
+## Training
+
+To run the training process, first one should install the python project on `venv`:
+```bash
+pip install poetry
+
+mkdir ~/.venvs; cd ~/.venvs
+python -m venv cattus
+
+pip install -e $CATTUS_TOP

```

Comment:
`-e` is for development
It allows you to keep using the development folder and pip makes sure that it is importable. Which is good because you want to keep *E*diting the folder.
So, we can just write to `pip install $CATTUS_TOP`, or even without a separate `git clone`:
`pip install git+https://github.com/poja/RL.git`

Reply:
I prefer you will do it if thats ok, i dont know it so good

Reply:
no problem, will do it tomorrow or friday